README.md

```diff
@@ -1 +1,146 @@
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Decode Morse Code <a name="about-project"></a>
+
+OOP School Library is a tool to record what books are in a library and who borrows them. The app allows you to add new students or teachers, add new books, and save records of who borrowed a given book and when.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby offcial website</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Person, student, and teacher classes
+- Student and Teacher classes inherits from Person class
+- Data preservation
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project, you must have installed and configured the following tools in your local environment:
+
+- [ ] Install Ruby
+
+### Setup
+
+**You must ensure that you have the postgreSQL installed and the server is started before continuing.**
+
+First clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/over-geek/OOP-School-Library.git
+  cd OPP-School-Library
+```
+### Install
+
+Install this project with:
+```sh
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  ruby person.rb
+  ruby student.rb
+  ruby teacher.rb
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Agyare Kissi Kenneth Kwame**
+
+- GitHub: [@githubhandle](https://github.com/over-geek)
+- Twitter: [@twitterhandle](https://twitter.com/KissiKenneth)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kenneth-agyare-kissi-673a01186/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- "Decorator" design patterns
+- basic UI
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ Required ] I think it would be nice if you ensure that all the links on your readme file are functioning as one would expect. Currently, when I click on the `issues` link it leads me to a `404` page not found error. Please see the example code below for a suggestion on how to solve this.

## Example code
```sh
![issues](https://github.com/over-geek/OOP-School-Library/issues)
```